assets/css/style.css

```diff
@@ -98,3 +99,178 @@ body {
   display: flex;
   justify-content: center;
 }
+
+.cards-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 320px);
+  gap: 26px;
+  place-content: center;
+}
+
+.product-card {
+  width: 320px;
+  border: solid 1px var(--very-light-black);
+  border-radius: 10px;
+}
+
+.product-card img {
+  width: 100%;
+  height: 200px;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  object-fit: cover;
+}
+
+.product-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 200px;
+}
+
+.product-info p {
+  margin: 0;
+  font-size: 25px;
+  font-weight: 600;
+  text-align: center;
+}
+
+.product-info button {
+  background-color: var(--green);
+  color: var(--white);
+  border-color: var(--green);
+  border-radius: 5px;
+  width: 122px;
+  height: 50px;
+  font-weight: 500;
+  font-size: var(--md);
+}
+
+.leng ul {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0;
+  margin: 0;
+}
+
+.leng ul li {
+  list-style: none;
+  border: solid 4px var(--gray);
+  border-radius: 7px;
+  margin: 5px;
+  background-color: var(--gray);
+}
+
+.cards h1 {
+  font-size: var(--lg);
+  font-weight: 700;
+  color: var(--light-black);
+  text-align: center;
+  margin: 0 20px 0 20px;
+}
+
+.dash {
+  display: flex;
+  justify-content: center;
+  margin: 25px 0 60px;
+}
+
+.about-me {
+  background-image: url(/assets/img/header-illsutration-mobile@2x.png);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  height: 300px;
+  margin-top: 40px;
+  margin-bottom: 60px;
+}
+
+.about-me h1 {
+  font-size: var(--lg);
+  font-weight: 700;
+  color: var(--light-black);
+  text-align: center;
+  margin: 0;
+}
+
+.about-me p {
+  font-size: var(--md);
+  text-align: center;
+  margin: 0 25px 0 25px;
+  font-weight: 400;
+}
```

Comment:
- [ ] Kindly make sure the about me description has the some `font-family`, `font-style`, `font-size`, `line-height`, and `color` as the one from [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988)